src/state/store/actions.ts

```diff
@@ -2,23 +2,26 @@
 
 import store from './_store';
 import { DialogState, ThemeState } from './types';
+import { setThemeMode, resetThemeMode, toggleThemeMode } from './theme.slice';
+import { dialogOpen, dialogClose } from './dialog.slice';
 
-export function closeDialog(): void {
-  store.dispatch({ type: 'dialog/close' });
+export function closeDialogAction(): void {
+  // @ts-ignore
+  store.dispatch(dialogClose());
 }
 
-export function openDialog(payload: Exclude<DialogState, null>): void {
-  store.dispatch({ type: 'dialog/open', payload });
+export function openDialogAction(payload: Exclude<DialogState, null>): void {
```

Comment:
Rather than `fooAction`, what do you think of `doFoo`?

The pattern is a bit more concise and I believe that's what we did at Olive.

Reply:
I can make that happen. I thought it might be nice to have `fooAction` to hopefully denote that it's a reducer action